addons/website_slides/static/src/js/slides_course_quiz.js

```diff
@@ -392,6 +458,7 @@ odoo.define('website_slides.quiz', function (require) {
                 var channelData = self._extractChannelData(slideData);
                 slideData.quizData = {
                     questions: self._extractQuestionsAndAnswers(),
+                    answerIds: slideData.answerIds || false,
```

Comment:
`|| false` again.

If your answerIds is an array, you should use `|| []` here, or just no default value if you always expect something to be passed.